content/docs/tasks/security/authn-policy/index.md

```diff
@@ -570,6 +570,96 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 401
 {{< /text >}}
 
+### End-user authentication with per-path requirements
+
+End-user authentication can be enabled or disabled based on request path. This is useful if you want to
+whitelist some paths from End-user authentication, like for paths used for health check or status report.
```

Comment:
Do not use "whitelist". 

Reword: 
This is useful if you disable authentication for some paths, for example, the path used for health check or status report.